next.config.js

```diff
@@ -3,21 +3,24 @@ const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
   experimental: {
-    appDir: true
+    appDir: true,
   },
   images: {
     formats: ["image/avif", "image/webp"],
     dangerouslyAllowSVG: true,
-    remotePatterns: [{ hostname: "cdn.sanity.io" }]
+    remotePatterns: [{ hostname: "cdn.sanity.io" }],
   },
   typescript: {
     // Set this to false if you want production builds to abort if there's type errors
-    ignoreBuildErrors: process.env.VERCEL_ENV === "production"
+    ignoreBuildErrors: process.env.VERCEL_ENV === "production",
   },
   eslint: {
     /// Set this to false if you want production builds to abort if there's lint errors
-    ignoreDuringBuilds: process.env.VERCEL_ENV === "production"
-  }
+    ignoreDuringBuilds: process.env.VERCEL_ENV === "production",
+  },
+  env: {
+    SECRET_ACCESS_TOKEN: process.env.SECRET_ACCESS_TOKEN,
+  },
```

Comment:
This should not be here. I believe the server files should already have access to env flags - https://nextjs.org/docs/app/building-your-application/configuring/environment-variables

Reply:
Resolved in commit:
57d8ed08202d661ed37f58b1d2ac624f5260d831